funnel/templates/project_layout.html.jinja2

```diff
@@ -0,0 +1,97 @@
+{% extends "layout.html.jinja2" %}
+{% set title_suffix = project.title_suffix %}
+{% from "baseframe/forms.html.jinja2" import ajaxform %}
+{% from "baseframe/components.html.jinja2" import faicon %}
+{%- from "macros.html.jinja2" import csrf_tag, project_header, project_details_header, proposal_card, project_share, video_thumbnail, registerblock, project_col_right %}
+{% block title %}{{ project.title }}{% endblock %}
+{% block description %}{{ project.tagline }}{% if project.tagline %} &ndash; {{ project.taglines }}{% endif %}{% endblock %}
```

Comment:
Why do we have `tagline` and `taglines`? It's confusing to have two similarly named attributes.

Reply:
Typo error. Fixed it